packages/types/src/exchange/error.ts

```diff
@@ -0,0 +1,13 @@
+export class ExchangeValidationError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'ValidationError'
+  }
```

Comment:
can you check if is it used anywhere, or just checking the `instance of`? if is checking for just the instance it can be dropped.